packages/runtime/src/account.ts

```diff
@@ -269,7 +269,7 @@ export class AccountStore implements AccountStoreI {
    * @param clearProgram application clear program
    * NOTE - approval and clear program must be the TEAL code as string
    */
-  addApp (appID: number, params: AppDeploymentFlags,
+  deployApp (appID: number, params: AppDeploymentFlags,
```

Comment:
We should do: 
1. rename
2. Create a function with old names and parameters, calling the renamed function and mark it [deprecated](https://stackoverflow.com/a/62991642/297094) , eg:
```
/// @deprecated `deployASA` should be used instead.
addAsset (assetId: number, name: string, asaDef: types.ASADef): modelsv2.AssetParams {
    return this.deployASA(assetId, name, asaDef)
```